Indexing/SubstitutionTree.hpp

```diff
@@ -56,10 +62,16 @@
 #include <iostream>
 #endif
 
+inline unsigned subsTreeQueryBank(unsigned n) 
+{ return 3 * n; }
+
+inline unsigned subsTreeResultBank(unsigned n) 
+{ return 3 * n + 1; }
+
+inline unsigned subsTreeNormResultBank(unsigned n) 
+{ return 3 * n + 2; }
+
 
-static constexpr int QUERY_BANK=0;
-static constexpr int RESULT_BANK=1;
-static constexpr int NORM_RESULT_BANK=3;
```

Comment:
So `NORM_RESULT_BANK` used to be 3, but now `subsTreeNormResultBank(0) == 2` - is this a problem?

Also, what happened to the "glue" bank we had at some point? Or did I misremember where that is?

Reply:
Yeah this is a legacy thing because there used to be an unused `NORM_QUERY_BANK = 2`. It doesn't make a difference that the concrete numbers are changed, but only that they are distinct.

The glue bank is in `RobSubstitution`.